src/cli/encrypt.go

```diff
@@ -71,7 +71,7 @@ func encryptCommand(root *rootCommand) *cobra.Command {
 			encryption.LogGroups(unencryptedGroups, logger)
 
 			// Dry run?
-			dryRun, _ := cmd.Flags().GetBool("dry-run")
+			dryRun := root.options.GetBool("dry-run")
```

Comment:
V tomto projekte je mozne zadat hodnotu aj cez ENV.
No to sa nesmie pristupovat priamo ku `Flags()` ale musi sa pouzit `Viper` kniznica, ... cez `Options` ... to je uz davno implementovane, ale niekde som to pouzival zle.